mapserver.h

```diff
@@ -2015,11 +2015,18 @@ void msPopulateTextSymbolForLabelAndString(textSymbolObj *ts, labelObj *l, char
 #endif /*SWIG*/
 
   /* Function prototypes, wrapable */
-  MS_DLL_EXPORT int msSaveImage(mapObj *map, imageObj *img, const char *filename);
-  MS_DLL_EXPORT void msFreeImage(imageObj *img);
-  MS_DLL_EXPORT int msSetup(void);
-  MS_DLL_EXPORT void msCleanup(void);
-  MS_DLL_EXPORT mapObj *msLoadMapFromString(char *buffer, char *new_mappath);
+  MS_DLL_EXPORT int msSaveImage(mapObj *map, imageObj *img, const char *filename); ///< Saves a map image to a file
```

Comment:
Some of the below makes long lines. using multiline /** */ syntax would be better and more common. That remark applies to maperror.h as well